api/src/opentrons/protocol_api/tasks.py

```diff
@@ -28,18 +28,21 @@ def created_at(self) -> datetime:
     @property
     @requires_version(2, 27)
     def done(self) -> bool:
-        """Whether the task is done."""
+        """Returns ``True`` if the task is done."""
         return self._core.is_done()
 
     @property
     @requires_version(2, 27)
     def started(self) -> bool:
-        """Whether the task has started."""
+        """WReturns ``True`` if the task is started."""
```

Comment:
```suggestion
        """Returns ``True`` if the task has started."""
```
Stray character. And I think "has" sounds more natural.